doc/whats_new.rst

```diff
@@ -42,6 +45,9 @@ Bug
 - Add option to plot the averaged dipole in `~hnn_core.viz.plot_dipole` when `dpl`
   is a list of dipoles, by `Huzi Cheng`_ in :gh:`475`.
 
+- Fix bug where :func:`~hnn_core.viz.plot_morphology` did not accurately
```

Comment:
I thought it wasn't allowed to change the shape with the API before?

Reply:
Yeah, I though it more had to do with the fact that user-defined changes to cell segment attributes weren't properly updated across other cell segment attributes.

Reply:
Since the current `Section.end_pts` don't reflect what is built in Neuron, the cell morphology plots have never shown the "true" shape of the cell being simulated. I think this is the most accurate description of the fix since the `end_pts` are really only used for plotting.

Reply:
Fair enough!